File: BetApp.Frontend/src/types/bet.ts
import { FormSchema } from "@/components/CreateBetForm";

export interface User {
  id: number;
  username: string;
}

export type Bet = {
  id: number;
  item: string;
  highestBid: number;
  userId: number;
  user: User;
}

export type CreateTodo = FormSchema;

File: Models/Bet.cs
namespace BetApp.Web.Models;

public class Bet
{
  public int Id { get; set; }

  public string Item { get; set; } = "";

  public int HighestBid { get; set; }

  public int UserId { get; set; }

  public User User { get; set; }

}



